scripts/pegin.sh

```diff
@@ -14,8 +15,8 @@ echo "Pegging in $PEG_IN_AMOUNT with confirmation time $CONFIRMATION_TIME"
 # random value only known to our client.
 ADDR="$($MINT_CLIENT peg-in-address)"
 
-# We send the amount we want to peg-in to this address
-TX_ID="$($BTC_CLIENT sendtoaddress $ADDR $PEG_IN_AMOUNT)"
+# We send the amount we want to peg-in to this address. notice that we have sats as default so we convert them to btc
+TX_ID="$($BTC_CLIENT sendtoaddress $ADDR $(expr $PEG_IN_AMOUNT / 1000))"
```

Comment:
uf that dosn't seem quite right xd

Reply:
You might need to use `bc` to deal with floating point numbers.

Reply:
yeah changed that